content/research/Benchmarking Honcho.md

```
@@ -0,0 +1,191 @@
+
+**TL;DR: Honcho achieves state-of-the-art performance across the LongMem, LoCoMo, and BEAM memory benchmarks. 90.4% on LongMem S (92.6% with Gemini 3 Pro), 89.9% on LoCoMo ([beating our previous score of 86.9%](https://blog.plasticlabs.ai/research/Introducing-Neuromancer-XR)), and top scores across all BEAM tests. We do so while maintaining competitive token efficiency. But recall tested in benchmarks which fit within a context window is no longer particularly meaningful. Beyond simple recall, Honcho reasons over memory and empowers frontier models to reason across more tokens than their context windows support. Go to [evals.honcho.dev](https://evals.honcho.dev) for charts and comparisons.**
+
+## 1. A primer on Honcho's architecture
+
+Read [Honcho's documentation](https://docs.honcho.dev) for a full understanding of how Honcho works, but a brief overview is important for understanding our benchmarking methodology and how Honcho achieves state-of-the-art results:
+
+- Honcho is a 'memory agent': it runs as a server which ingests messages, reasons, and exposes a variety of query endpoints.
+
+- When a message or batch of messages is stored in Honcho, it uses small fine-tuned models to capture all latent information and save it as a 'Representation' of the author[^4].
+
+- In the background, Honcho regularly 'dreams'[^5] across ingested messages and prior reasoning to make deductions and reach further conclusions. 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)